sdk/file_sdk/Common/file_format.cpp

```diff
@@ -0,0 +1,25 @@
+#include "file_format.h"
+
+namespace mip {
+namespace file {
+
+FileFormat::FileFormat(
+    std::shared_ptr<IStream> file,
+    const string& extension)
+  : mFile(file),
+    mExtension(extension)

```

Comment:
mTags()

---
In reply to: [134442643](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134442643) [](ancestors = 134442643,134442403,134442270,134435631)